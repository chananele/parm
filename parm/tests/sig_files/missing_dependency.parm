---
name: missing_dep
imports:
  - start
exports:
  - end
method: find_single
pattern: |
  start: b @:end
...
