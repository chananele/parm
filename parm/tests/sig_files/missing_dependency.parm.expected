---
name: missing_dep
result: not run
errors:
  - unresolved import [start]
...


